cider/featurizer.py

```diff
@@ -417,6 +418,12 @@ def mobilemoney_features(self) -> None:
         if self.ds.mobilemoney is None:
             raise ValueError('Mobile money file must be loaded to calculate mobile money features.')
         print('Calculating mobile money features...')
+        
+        # create dummy columns for missing field (resulting features will all take value N/A, so be
+        #  ignored during model-fitting). This logic should eventually be refined.
+        for col_name in ['sender_balance_before', 'sender_balance_after', 'recipient_balance_before', 'recipient_balance_after']:
+            if col_name not in self.ds.mobilemoney.columns:
+                self.ds.mobilemoney = self.ds.mobilemoney.withColumn(col_name, lit(None).cast(DoubleType()))
```

Comment:
can you confirm how a `null` in a `Double`-typed column is serialized in Parquet and CSV? a null mobilemoney amount is semantically different from a zero, and i want to make sure we're not forcing missing values to be 0 at this stage

Reply:
neat, thanks!